### DISCUSSION TITLE
MessageBox调用异常

### DISCUSSION THREAD

**Discussion by yzy945** [irihitech/Ursa.Avalonia#251] [Q&A] [answered]
调用显示MessageBox时如图只有外框，框内为透明，能看到后面窗体
![2024-05-21_111958](https://github.com/irihitech/Ursa.Avalonia/assets/37538175/e4da9fff-1ae4-446c-a200-bdb5b6c2aed4)

codebehind调用，没写mvvm
`MessageBox.ShowAsync("_messdddddage", "", icon: MessageBoxIcon.Error, button: MessageBoxButton.OK);`

**Comment by zdpcdt** [comment, marked as answer]
你要引用一下Semi的样式，在App.axmal文件下添加：

```axaml
<Application.Styles>
    <StyleInclude Source="avares://Semi.Avalonia/Themes/Index.axaml" />
    <u-semi:SemiTheme Locale="zh-CN" />
</Application.Styles>
```
